contracts/Medianizer.sol

```diff
@@ -127,11 +102,11 @@ contract Medianizer is DSMath {
         if (ctr % 2 == 0) {
             uint128 val1 = uint128(wuts[(ctr / 2) - 1]);
             uint128 val2 = uint128(wuts[ctr / 2]);
-            value = bytes32(wdiv(hadd(val1, val2), 2 ether));
+            value = bytes32((val1 + val2) / 2);
```

Comment:
This simple code is okay `(val1 + val2) / 2` because all price values are cast to `uint128` which ensures that `val1 + val2) will not be greater than 2^256 - 1.